apps/wallet-mobile/src/features/Send/common/useSendReceiver.tsx

```diff
@@ -25,6 +25,16 @@ export const useSendReceiver = () => {
       enabled: false,
     },
   )
+
+  const isNotResolvedDomain = React.useMemo(
+    () =>
+      !isUnsupportedDomain &&
+      !isResolvingAddressess &&
+      cryptoAddresses.length > 0 &&
+      cryptoAddresses.every(({error}) => error !== null),
+    [cryptoAddresses, isResolvingAddressess, isUnsupportedDomain],
+  )
```

Comment:
to cover https://emurgo.atlassian.net/browse/YOMO-1072

Reply:
important consideration here https://github.com/Emurgo/yoroi/pull/2959#discussion_r1440987558